Dockerfile

```diff
@@ -0,0 +1,66 @@
+
+### 1. Get Linux
+FROM ubuntu:18.04
+
+ENV ROBOT 1.5.0
+ARG ROBOT_JAR=https://github.com/ontodev/robot/releases/download/$ROBOT/robot.jar
+ENV ROBOT_JAR ${ROBOT_JAR}
+ENV DOSDPVERSION 0.13.1
+ENV JENA 3.12.0
+ENV BGR 1.4
+ENV KBOT 1.9.1
+
+
+### 2. Get Java and all required system libraries
+RUN apt-get update && apt-get upgrade -y \
+ && apt-get install -y software-properties-common \
+  build-essential git \
+  openjdk-8-jre openjdk-8-jdk
+
+RUN apt-get update \
+    && apt-get install -y make curl git wget tar
```

Comment:
This needlessly runs another `apt-get update` (it was just run) and the separate`RUN` instruction introduces another layer in the filesystem, probably needlessly.